README.md

```diff
@@ -1,14 +1,32 @@
 # melike2d/lavalink
 
-> A version of lavalink that includes [lavadsp](https://github.com/natanbc/lavadsp) filters.
-
-- **Working Dockerfile**: <https://github.com/MeLike2D/lavalink/blob/dev/LavalinkServer/docker/Dockerfile>
+#### A version of [lavalink](https://github.com/freyacodes/lavalink) that includes [lavadsp](https://github.com/natanbc/lavadsp) filters to spice up your audio.
 
 ## Warning
 
-If you use ip-rotation make sure to use the updated application.yml structure, or you might not get working ip rotation. <https://github.com/melike2d/lavalink/blob/dev/LavalinkServer/application.yml.example>
+If you use IP Rotation make sure to use the updated application.yml structure, or you won't be geting a working IPV6 rotation. <https://github.com/melike2d/lavalink/blob/dev/LavalinkServer/application.yml.example>
```

Comment:
"If you use ip rotation make sure to use the updated application.yml file structure, or you might be at risk to ip bans."